site/content/docs/mdx.md

```diff
@@ -0,0 +1,288 @@
+import { Callout } from '../components/TempCallout.jsx'
+
+# MDX
+
+Flowershow parses all of your Markdown files as MDX. This means you not only can write your content using good old Markdown, but also extend it with JSX, JavaScript expressions, and `import` and `export` statements. It's basically Markdown on steroids 😀💪!
+
+Let's see what exactly is MDX and what's so cool about it!
+
+<Callout>
```

Comment:
> One major question (to address going forward) is about whether we can locate our components in `site/content` vs in `templates/default` as per my one major comment

At the moment, having components inside `/content` folder doesn't work. It will throw an error related to having mixed content there. But maybe this can be set somehow in Contentlayer.

> 
> > triangular_flag_on_post where is the `components` folders? This is crucial info for someone following along (is it relative to their `content` directory or where?

Totally agree, I'll make it more explicit.